server/contracts.go

```diff
@@ -0,0 +1,63 @@
+package server
+
+import (
+	"fmt"
+	"io/ioutil"
+	"regexp"
+
+	emulator "github.com/onflow/flow-emulator"
+	"github.com/onflow/flow-go-sdk"
+	"github.com/onflow/flow-go-sdk/templates"
+	"github.com/onflow/flow-nft/lib/go/contracts"
+)
+
+var ftAddress = flow.HexToAddress("ee82856bf20e2aa6")
+var baseContractsPath = "./server/contracts/"
+
+// deployContracts is meant to be a place where you can load up existing contracts
+// NFTStorefront is an example of how you can do this
+func deployContracts(conf *Config, b *emulator.Blockchain) map[string]flow.Address {
+	addresses := make(map[string]flow.Address)
+
+	nftAddress, _ := deployContract("NonFungibleToken", contracts.NonFungibleToken(), b, addresses)
+	deployContract("ExampleNFT", contracts.ExampleNFT(nftAddress.Hex()), b, addresses)
+
+	nftStorefront := loadContract("NFTStorefront.cdc", map[string]flow.Address{
+		"FungibleToken":    ftAddress,
+		"NonFungibleToken": nftAddress,
+	})
+	deployContract("NFTStorefront", nftStorefront, b, addresses)
+
+	return addresses
+}
+
+func loadContract(name string, replacements map[string]flow.Address) []byte {
+	code := string(readFile(baseContractsPath + name))
```

Comment:
I believe it would be better to use filepath package join function https://pkg.go.dev/path/filepath#Join